docs/RELEASE_POLICY.md

```diff
@@ -120,7 +120,7 @@ The support period of deprecated API elements is tied to
 ## Developer preview features
 
 A feature is "in" developer preview if it’s marked as such in the
-[Firecracker roadmap](https://github.com/firecracker-microvm/firecracker/projects/13)
+[Firecracker roadmap](https://github.com/firecracker-microvm/firecracker/projects/42)
```

Comment:
I see... so huge pages will not work without correct links..

Reply:
Yeah, whenever you enable hugetlbfs, firecracker will first clone its git repository and then refuse to boot the vm if it finds any invalid links